components/runtime/wine/patches/03-ntdll-virtual.patch

```diff
@@ -0,0 +1,31 @@
+--- wine-4.4.orig/dlls/ntdll/virtual.c	2019-03-15 23:18:31.000000000 +0000
++++ wine-4.4/dlls/ntdll/virtual.c	2019-07-11 23:36:02.221283140 +0000
+@@ -1853,8 +1853,10 @@
+  */
+ void virtual_get_system_info( SYSTEM_BASIC_INFORMATION *info )
+ {
+-#ifdef HAVE_SYSINFO
++#if defined(HAVE_SYSINFO) && !defined(__sun)
```

Comment:
Common idiom is 
```
#if defined(__sun) 
...
#else
...
#endif
``

Reply:
yeah our `sysinfo(2)` somehow gets picked up by autoconf as being "correct", despite the *very* different prototype...

Reply:
fixed. patch has the common idiom now.